### DISCUSSION TITLE
v0.11.0-beta2

### DISCUSSION THREAD

**Discussion by weltenwort** [weltenwort/frigate-synology-dsm7#44] [Announcements]
:warning: **ATTENTION:** this minor version release changes the database structure in a way that prevents downgrades to prior versions

## Significant Changes
* Update and fix build for 0.11.0 beta 2 by @deviant77 in https://github.com/weltenwort/frigate-synology-dsm7/pull/43

## CI Changes
* Bump docker/login-action from 1.13.0 to 1.14.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/23
* Bump docker/login-action from 1.14.0 to 1.14.1 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/24
* Bump actions/checkout from 2 to 3 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/25
* Bump sigstore/cosign-installer from 1.4.1 to 2.0.1 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/21
* Bump docker/build-push-action from 2.9.0 to 2.10.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/27
* Bump docker/metadata-action from 3.6.2 to 3.7.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/28
* Bump sigstore/cosign-installer from 2.0.1 to 2.2.1 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/30
* Bump sigstore/cosign-installer from 2.2.1 to 2.3.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/31
* Bump docker/setup-buildx-action from 79abd3f86f79a9d68a23c75a09a9a85889262adf to 1.7.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/32
* Bump docker/login-action from 1.14.1 to 2 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/37
* Bump docker/metadata-action from 3.7.0 to 4.0.1 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/36
* Bump docker/setup-buildx-action from 1.7.0 to 2 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/35
* Bump docker/build-push-action from 2.10.0 to 3 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/34
* Enable manually running workflow actions by @deviant77 in https://github.com/weltenwort/frigate-synology-dsm7/pull/39
* Bump sigstore/cosign-installer from 2.3.0 to 2.4.0 by @dependabot in https://github.com/weltenwort/frigate-synology-dsm7/pull/40

**Full Changelog**: https://github.com/weltenwort/frigate-synology-dsm7/compare/v0.10.0-amd64...v0.11.0-beta2

<hr /><em>This discussion was created from the release <a href='https://github.com/weltenwort/frigate-synology-dsm7/releases/tag/v0.11.0-beta2'>v0.11.0-beta2</a>.</em>

**Comment by deviant77** [comment]
Be aware that there are significant changes to the database schema in 0.11.0, so you'll need to restore a backup of the database if you want to go back to 0.10.0.

**Reply by weltenwort** [reply]
Good pointer, thanks. I'll add a warning to the release description.
